packages/web-react/src/components/UNSTABLE_Slider/__tests__/UNSTABLE_Slider.test.tsx

```diff
@@ -0,0 +1,52 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import UNSTABLE_Slider from '../UNSTABLE_Slider';
+
+describe('UNSTABLE_Slider', () => {
+  const id = 'slider-test';
+  const label = 'Slider';
+  const onChange = () => {};
+  const defaultValue = 10;
+
+  stylePropsTest((props) => <UNSTABLE_Slider id={id} {...props} data-testid="slider-test" />, 'slider-test');
+
+  restPropsTest((props) => <UNSTABLE_Slider id={id} {...props} />, 'div');
+
+  it('should render slider', () => {
+    render(<UNSTABLE_Slider id={id} value={defaultValue} onChange={onChange} label={label} />);
+
+    const sliderElement = screen.getByRole('slider');
+
+    expect(screen.getByText(label)).toBeInTheDocument();
+    expect(sliderElement).toBeInTheDocument();
```

Comment:
Isn't this test implicitly true when `screen.getByRole('slider')` does not fail?

Reply:
i think i don't understand the question

Reply:
we discussed it and we leave it like this